### PR TITLE
[add] Instructions to use the docker appbuilder in README

### PR DESCRIPTION
**Pull request by lpascal-ledger**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,25 @@ This is a boilerplate application which can be forked to start a new project for
...
+You can download it from the ghcr.io docker repository:
+
+```shell
+docker pull ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder-full
+```
+
+You can then enter this development environment by executing the following command from the directory of the application `git` repository:
+
+```shell
+docker run --rm -ti  -v "$(realpath .):/app" ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder-full
```
Comment (xchapron-ledger) [inline README.md:19]: Maybe adding `--user "$(id -u)":"$(id -g)"` can be a good idea.
It helps when docker needs to be run under `sudo`.

Comment (lpascal-ledger) [reply README.md:19]: Indeed!

**Review by xchapron-ledger — COMMENTED** [review]
I think that this helps, but I think we should go further and propose a simple script that can be easily called and that would handle all the environement. Something like our Quick build tool or something like some external dev do: https://github.com/LedgerHQ/app-iota/pull/6/files/867bb78859234cb304888b7db046881986e7baf3#diff-4d2a8eefdf2a9783512a35da4dc7676a66404b6f3826a8af9aad038722da6823

**Comment by lpascal-ledger** [comment]
@xchapron-ledger I think this goes beyond this PR and maybe this repo even.

File: tests/requirements.txt [hunk 2]
```diff
@@ -1,6 +1,7 @@
 pytest
 ragger[speculos]>=1.2.0
 ragger[ledgerwallet]>=1.2.0
+protobuf==3.20.0
```
Comment (fbeutin-ledger) [inline tests/requirements.txt:4]: Why do we need to tag protobuf here ?
It's not boilerplate that depends on it but ledgerwallet, and ledgerwallet now specifies the correct version?
Not deployed though, maybe that's why

**Review by fbeutin-ledger — APPROVED** [review]
I also think that we should go further than just the docker and provide a builder script, however I also agree that it's outside the scope of this PR, and that this PR is a net improvement so I suggest merging this one and keeping the builder topic for later

File: tests/requirements.txt (hunk 2 above)
Comment (lpascal-ledger) [reply tests/requirements.txt]: Removed this version

**Pull request merged**